tests/framework/info.go

```diff
@@ -53,3 +56,28 @@ func GetEvents(rm ResourceManager, namespace string) string {
 	}
 	return returnEvents
 }
+
+// GetBuildInfo returns the build information.
+func GetBuildInfo() (commitHash string, commitTime string, dirtyBuild string) {
+	commitHash = "unknown"
+	commitTime = "unknown"
+	dirtyBuild = "unknown"
+
+	info, ok := debug.ReadBuildInfo()
+	if !ok {
+		return
+	}
+
+	for _, kv := range info.Settings {
+		switch kv.Key {
+		case "vcs.revision":
+			commitHash = kv.Value
+		case "vcs.time":
+			commitTime = kv.Value
+		case "vcs.modified":
+			dirtyBuild = kv.Value
+		}
+	}
+
+	return
```

Comment:
Why do we return here?

Reply:
https://pkg.go.dev/golang.org/x/tools/internal/typesinternal#MissingReturn

Reply:
😦 👍